.github/CODEOWNERS

```diff
@@ -108,20 +108,20 @@ core/scripts/gateway @smartcontractkit/functions
 /.github/workflows/automation-load-tests.yml @smartcontractkit/keepers
 /.github/workflows/automation-nightly-tests.yml @smartcontractkit/keepers
 
-/core/chainlink.Dockerfile @smartcontractkit/prodsec-public
+/core/chainlink.Dockerfile @smartcontractkit/prodsec-public @smartcontractkit/foundations
 
 # Dependencies
-contracts/scripts/requirements.txt @smartcontractkit/prodsec-public
-.tool-versions @smartcontractkit/prodsec-public
-.nvmrc @smartcontractkit/prodsec-public
-contracts/package.json @smartcontractkit/prodsec-public
-contracts/pnpm.lock @smartcontractkit/prodsec-public
+contracts/scripts/requirements.txt @smartcontractkit/prodsec-public @smartcontractkit/foundations
+.tool-versions @smartcontractkit/prodsec-public @smartcontractkit/foundations
+.nvmrc @smartcontractkit/prodsec-public @smartcontractkit/foundations
+contracts/package.json @smartcontractkit/prodsec-public @smartcontractkit/foundations
+contracts/pnpm.lock @smartcontractkit/prodsec-public @smartcontractkit/foundations
```

Comment:
very often, @smartcontractkit/releng is asked for help with packages and artifacts - IMO, it's worth adding the RE team as a code owner for /contracts, Dockerfiles, and go.mods. etc.
@chainchad, what are your thoughts?

Reply:
Agreed for Dockerfile's. For contracts, at the very least, `@smartcontractkit/releng` should be on:

```
contracts/package.json
package.json
```

As these are modified when incrementing a verison for a new release.

Reply:
Here's what I think we should add:

```
/.goreleaser.*.yaml @smartcontractkit/releng
**/*.Dockerfile @smartcontractkit/releng @smartcontractkit/foundations 
**/Dockerfile @smartcontractkit/releng @smartcontractkit/foundations
**/package.json @smartcontractkit/releng @smartcontractkit/prodsec-public
```